compute_tools/src/compute.rs

```diff
@@ -402,8 +404,12 @@ impl ComputeNode {
         let mut client = Client::connect(self.connstr.as_str(), NoTls)?;
         self.pg_reload_conf(&mut client)?;
 
+        // Disable DDL forwarding
```

Comment:
```suggestion
```
this is now redundant with the longer comment two lines down

Reply:
Good catch, thanks